### DISCUSSION TITLE
About the datasets supported by OpenCompass

### DISCUSSION THREAD

**Discussion by niexufei** [open-compass/opencompass#250] [Q&A]
1. Does OpenCompass support datasets for Chinese-English translation tasks? Regarding language, I noticed the Flores dataset, but is its focus not on translation between Chinese and English? 
2. Does OpenCompass support datasets for evaluating model safety? For example, evaluating whether models can output safe content in terms of ethics, morality, and politics. Is there a plan to support datasets in this aspect?
Thanks.

**Comment by tonysy** [comment]
Thanks for the insightful suggestions.

1. We are working on supporting more Chinese-English translation tasks, and we also build some pure-Chinese datasets. More information will be updated soon.
2. We have already supported some safety datasets in codebase.
like CrowsPairs, RealToxicPrompt, etc.

Welcome to contribute more datasets or provide your insightful suggestions.

**Comment by yingfhu** [comment]
Crowspairs, CivilComments, JigsawMultilingual, TruthfulQA, RealToxicPrompt, these datasets are supported currently, you are welcomed to try them.

**Comment by tonysy** [comment]
Also , we have launched community collobration plan, see more information in https://github.com/InternLM/opencompass/issues/248

**Comment by niexufei** [comment]
> Thanks for the insightful suggestions.
> 
> 1. We are working on supporting more Chinese-English translation tasks, and we also build some pure-Chinese datasets. More information will be updated soon.
> 2. We have already supported some safety datasets in codebase.
>    like CrowsPairs, RealToxicPrompt, etc.
> 
> Welcome to contribute more datasets or provide your insightful suggestions.

Thanks for your reply.
I have found that Crowspairs, CivilComments, JigsawMultilingual, TruthfulQA, and RealToxicPrompt are all English datasets related to safety. Do you have plans to support Chinese datasets as well?
